bin/near-cli.js

```diff
@@ -170,6 +171,22 @@ yargs // eslint-disable-line
         desc: 'Unique identifier for the account',
         type: 'string',
     })
+
+    .option('walletUrl', {
+        desc: 'Website for NEAR Wallet',
+        type: 'string',
+        hidden: true
+    })
+    .option('contractName', {
+        desc: 'Account name of contract',
+        type: 'string',
+        hidden: true
+    })
+    .option('masterAccount', {
+        desc: 'Master account used when create new accounts',
```

Comment:
Please change "used when create" to "used to create" or "used when creating"

Reply:
done